tests/components/zwave_js/test_update.py

```diff
@@ -432,73 +432,85 @@ async def test_update_entity_progress_multiple(
     attrs = state.attributes
     assert attrs[ATTR_IN_PROGRESS] is True
 
-    event = Event(
-        type="firmware update progress",
-        data={
-            "source": "node",
-            "event": "firmware update progress",
-            "nodeId": node.node_id,
-            "sentFragments": 1,
-            "totalFragments": 20,
-        },
+    node.receive_event(
+        Event(
+            type="firmware update progress",
+            data={
+                "source": "node",
+                "event": "firmware update progress",
+                "nodeId": node.node_id,
+                "sentFragments": 1,
+                "totalFragments": 20,
+            },
+        )
     )
-    node.receive_event(event)
+
+    # Block so HA can do its thing
+    await asyncio.sleep(0)
```

Comment:
We can use `hass.async_block_till_done`.

Reply:
that would cause the test to hang when I tried it. I assumed because it was waiting until the service call was done? I tried to remove the blocking=True from the service call but that didn't help

Reply:
Then we should probable create the task with `asyncio.create` task instead. We await the task in the test anyway.

Reply:
the task for the service? Just tried that and still hanging after switching to async_block_till_done

Reply:
Ok, yeah, the service call adds a tracked task.